guide/designing-products/user-research-resources.md

```diff
@@ -37,29 +37,53 @@ Below is a collection of published user research findings around Bitcoin, from a
 ## Research papers
 
 ###### [Non-adoption of crypto-assets: Exploring the role of trust, self-efficacy, and risk](https://voskart.de/pdf/crypto_adoption.pdf)
+
+*A survey of 404 users and non-users of Bitcoin about the roles of trust, self-efficacy, and risk on adoption intention.*
+
 Published in May 2021 by Artemij Voskobojnikov, Svetlana Abramova, Konstantin Beznosov, and Rainer Böhme.
 
 ###### [Bits Under the Mattress: Understanding Different Risk Perceptions and Security Behaviors of Crypto-Asset Users](https://voskart.de/pdf/bits_under_mattress.pdf)
+
+*A survey-based exploration of different security behaviors among three Bitcoin user archetypes: Cypherpunks, hodlers, and rookies. This study demonstrates that cypherpunks tend to use self-managed security solutions, whereas hodlers and rookies use either custodial solutions or non-custodial wallets.*
+
 Published in February 2021 by Svetlana Abramova, Artemij Voskobojnikov, Konstantin Beznosov, and Rainer Böhme.
 
 ###### [The U in Crypto Stands for Usable: An Empirical Study of User Experience with Mobile Cryptocurrency Wallets](https://voskart.de/pdf/u_in_crypto.pdf)
+
+*In a corpus of 45,821 app reviews of the top five mobile cryptocurrency wallets, 6,859 reviews were identified and qualitatively analyzed pertaining to the user experience with those wallets. The analysis suggests that both new and experienced users struggle with general and domain-specific UX issues that, aside from frustration and disengagement, might lead to dangerous errors and irreversible monetary losses. We reveal shortcomings of current wallet UX as well as users’ misconceptions, some of which can be traced back to a reliance on their understanding of conventional payment systems.*
+
 Published in January 2021 by Artemij Voskobojnikov, Oliver Wiese, Masoud Mehrabi Koushki, Volker Roth, and Konstantin Beznosov.
 
 ###### [User Perception of Bitcoin Usability and Security across Novice Users](https://www.researchgate.net/publication/331286661_User_Perception_of_Bitcoin_Usability_and_Security_across_Novice_Users)
+
+*This paper investigates users’ perceptions and experiences of an anonymous digital payment system (Bitcoin) and its influence on users in terms of usability and security in comparison to other non-anonymous payment systems such as credit/debit cards. This paper identifies users’ perceptual differences in terms of usability and security. We conclude that Bitcoin, as a crypto-currency, is still in its infancy and requires user education and a new way of thinking. The study recommends developing users’ mental models to deepen developers’ understanding of anonymous digital payment technology and improve user-centred design.*
```

Comment:
```suggestion
*A survey of undergraduate students at Keele University concluding that Bitcoin was still in its infancy at the time of writing and required user education and a new way of thinking. The study recommends user-centered design to develop users’ mental models and deepen their understanding of digital payment technology.*
```
Shortened. Also highlighted that the survey was only taken by undergraduate students at 1 university, which probably makes it not representative globally.